HealthcareSystem/Backend/Service/NotificationSurveyAndFeedback/FeedbackService.cs

```diff
@@ -21,51 +21,31 @@ public FeedbackService(IFeedbackRepository feedbackRepository)
         {
             _feedbackRepository = feedbackRepository;
         }
-        /// <summary>
-        /// /adding new feedback to database
-        /// </summary>
-        /// <param name="feedback">an object to be added to the database</param>
         public void AddFeedback(Feedback feedback)
         {
+            if (feedback == null)
+                throw new BadRequestException("Please, write a comment to send feedback.");
             _feedbackRepository.AddFeedback(feedback);
         }
-        /// <summary>
-        /// /updating feedbacks status (property: IsPublished) to published
-        /// </summary>
-        /// <param name="id">id of the object to be changed</param>
         public void PublishFeedback(int id)
         {
             Feedback feedback = GetFeedbackById(id);
+            if (!feedback.IsAllowedToPublish)
+                throw new BadRequestException("Feedback is not allowed to publish.");
             feedback.IsPublished = true;
             _feedbackRepository.UpdateFeedback(feedback);
         }
-        /// <summary>
-        /// /getting all published feedbacks
-        /// </summary>
-        /// <returns>list of published feedbacks</returns>
         public List<Feedback> GetPublishedFeedbacks()
         {
             return _feedbackRepository.GetPublishedFeedbacks();
         }
-        /// <summary>
-        /// /getting all unpublished feedbacks
-        /// </summary>
-        /// <returns>list of unpublished feedbacks</returns>
         public List<Feedback> GetUnpublishedFeedbacks()
         {
             return _feedbackRepository.GetUnpublishedFeedbacks();
         }
-        /// <summary>
-        /// /getting feedback by id
-        /// </summary>
-        /// <param name="id">id of the wanted object</param>
-        /// <returns>object type Feedback</returns>
         public Feedback GetFeedbackById(int id)
         {
-            Feedback feedback = _feedbackRepository.GetFeedbackById(id);
-            if (feedback == null)
-                throw new NotFoundException("Feeback with id=" + id + " doesn't exist in database.");
-            return feedback;
+            return _feedbackRepository.GetFeedbackById(id);
```

Comment:
@kmilena98 Why you deleted @jelenabudisa documentation?

@sladjana-savkovic check this, is this okay for your team? ( I think on throwing exception there )?